filesystems/android_sparse/README.md

```diff
@@ -0,0 +1,22 @@
+<!--
+SPDX-FileCopyrightText: 2021 Armijn Hemel <armijn+github@tjaldur.nl>
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+## android_sparse.img
+
+An Android sparse image, that expands to a 4k raw image filled with zero bytes.
+
+It was created using the following commands:
+
+    $ dd if=/dev/zero of=android_sparse.raw bs=4k count=1
```

Comment:
```suggestion
An Android sparse image, that expands to a 4 KiB raw image filled with zero bytes.

It was created using the following commands:

    $ dd if=/dev/zero of=android_sparse.raw bs=4K count=1
```

This eliminates any confusion whether kilo- or kibi- prefix is intended. Also `dd --help` is a bit unclear in what lowercase `k` suffix means:

<pre><code>N and BYTES may be followed by the following multiplicative suffixes:
c=1, w=2, b=512, <strong>kB=1000, K=1024</strong>, MB=1000*1000, M=1024*1024, xM=M,
GB=1000*1000*1000, G=1024*1024*1024, and so on for T, P, E, Z, Y.
Binary prefixes can be used, too: KiB=K, MiB=M, and so on.</code></pre>

After testing, however, it's clear that the `K=1024` option is used.